### ISSUE TITLE
GaussianBlurEffect.BlurAmount doc is confusing

### ISSUE THREAD

**Issue by asklar** [microsoft/Win2D#789]
The doc for [`GaussianBlurEffect.BlurAmount`](https://microsoft.github.io/Win2D/html/P_Microsoft_Graphics_Canvas_Effects_GaussianBlurEffect_BlurAmount.htm) is slightly confusing - it is supposed to describe blur amount, but then goes on to relate standard deviation and radius. How are radius and amount (or deviation and amount) related? 
We are trying to match other iOS and Android behaviors where they use a formula for the boxSize (which we believe represent amount in Win2D?) that relates to the radius as per https://github.com/facebook/react-native/blob/master/Libraries/Image/RCTImageBlurUtils.m#L48
uint32_t boxSize = floor((radius * imageScale * 3 * sqrt(2 * M_PI) / 4 + 0.5) / 2);

but this differs from the "divide by 3" rule, so... help @shawnhar ? :)
CC @marlenecota @appden

**Comment by asklar** [comment]
@lyahdav @appden : Shawn and team mentioned that it's unclear what the boxSize on iOS is supposed to be (are they using a box filter instead of a gaussian blur?). The BlurAmount parameter in Win2D is the standard deviation. What does the iOS implementation do?

**Comment by appden** [comment]
Yes exactly, the iOS/macOS implementation is using a box filter on GPU with the Apple Accelerate framework (see [Blurring an Image](https://developer.apple.com/documentation/accelerate/blurring_an_image)). The SVG spec [suggests](https://www.w3.org/TR/SVG11/filters.html#feGaussianBlurElement) this conversion from Guassian blur radius by applying three repeated convolutions, which is also referenced by the Wikipedia article on [Box blur](https://en.wikipedia.org/wiki/Box_blur):
> Box blurs are frequently used to approximate a Gaussian blur.[2] By the central limit theorem, repeated application of a box blur will approximate a Gaussian blur.[3]
Since the Win2D API is already specified Gaussian blur std deviation, maybe your "divide by 3" rule is correct and the difference in blur effect lies with how DPI is handled?
